jib-core/src/main/java/com/google/cloud/tools/jib/configuration/DockerHealthCheck.java

```diff
@@ -29,13 +30,15 @@
   /** Builds the immutable {@link DockerHealthCheck}. */
   public static class Builder {
 
-    @Nullable private final ImmutableList<String> command;
+    private final ImmutableList<String> command;
     @Nullable private Duration interval;
     @Nullable private Duration timeout;
     @Nullable private Duration startPeriod;
     @Nullable private Integer retries;
 
-    private Builder(@Nullable ImmutableList<String> command) {
+    private Builder(ImmutableList<String> command) {
+      Preconditions.checkArgument(command.size() > 1, "command must not be empty");
+      Preconditions.checkArgument(!command.contains(""), "command must not contain empty tokens");
```

Comment:
Should we be checking if command.stream().anyMatch(str -> str.trim().isEmpty) or something? Not just against the empty string but the whitespace string?